mappings/net/minecraft/client/GlExtensionHandler.mapping

```diff
@@ -0,0 +1,112 @@
+CLASS none/cbf net/minecraft/client/GlExtensionHandler
+	CLASS none/cbf$a FBOMode
+		FIELD a BASE Lnone/cbf$a;
+		FIELD b ARB Lnone/cbf$a;
+		FIELD c EXT Lnone/cbf$a;
+	FIELD A GL_SOURCE1_RGB I
+	FIELD B GL_SOURCE2_RGB I
+	FIELD C GL_OPERAND0_RGB I
+	FIELD D GL_OPERAND1_RGB I
+	FIELD E GL_OPERAND2_RGB I
+	FIELD F GL_COMBINE_ALPHA I
+	FIELD G GL_SOURCE0_ALPHA I
+	FIELD H GL_SOURCE1_ALPHA I
+	FIELD I GL_SOURCE2_ALPHA I
+	FIELD J GL_OPERAND0_ALPHA I
+	FIELD K GL_OPERAND1_ALPHA I
+	FIELD L GL_OPERAND2_ALPHA I
+	FIELD M extVersionBlendFuncSeparate Z
+	FIELD N hasGL21 Z
```

Comment:
Is there any harm in them having the same name considering that it's supported? Furthermore I can't seem to come up with a better name.